iceoryx_examples/iceperf/uds.cpp

```diff
@@ -90,7 +90,7 @@ void UDS::initFollower() noexcept
 void UDS::init() noexcept
 {
     // init subscriber
-    iox::posix::posixCall(socket)(AF_LOCAL, SOCK_DGRAM, 0)
```

Comment:
If we do not use this abstraction then this example will not run in windows. In general I would agree but here it is just a parameter forwarding - a teeny tiny overhead. Could you live with it or do you think that this would effect the outcome of the benchmark in a measurable way?

Reply:
Well, I don't mind. You have to ask budrus.

Reply:
@budrus said its alright!